server/container/guice/blob-export-guice/src/test/java/org/apache/james/modules/LinshareGuiceExtension.java

```diff
@@ -55,4 +55,9 @@ public Module getModule() {
             }
         );
     }
+
+    public void stopDockerLinshare() {
```

Comment:
Not possible to put the stop in an `AfterAll` callback here? So that you don't need to repeat it in the test classes it's used?

Reply:
if I put stopping in after all of `LinshareGuiceExtension`, then if there is a test class in the future doesn't want to restart linshare docker - I have to write another guice extension to not stop docker after all

Reply:
just anticipating

Reply:
In the contrary case we might just forget again to stop it if we need to... well either way is fine I guess